Starting with the switch to GCC 4.8.3, we observed this test failure.
Changing "34" to "134" means that we expect the date to be parsed as
"2034", not "1934", which seems consistent with the line above.

Reported at <http://bugs.icu-project.org/trac/ticket/10960>.

--- icu/source/test/intltest/dtfmttst.cpp	2014-06-16 10:35:46.000000000 +0200
+++ icu/source/test/intltest/dtfmttst.cpp	2014-06-16 10:35:52.000000000 +0200
@@ -1129,7 +1129,7 @@ DateFormatTest::TestTwoDigitYear()
         return;
     }
     parse2DigitYear(fmt, "5/6/17", date(117, UCAL_JUNE, 5));
-    parse2DigitYear(fmt, "4/6/34", date(34, UCAL_JUNE, 4));
+    parse2DigitYear(fmt, "4/6/34", date(134, UCAL_JUNE, 4));
 }
 
 // -------------------------------------
